RLTest/redis_std.py

```diff
@@ -3,24 +3,42 @@
 import subprocess
 import sys
 import os
-import platform
+import uuid
 from .utils import Colors, wait_for_conn
 
 
-MASTER = 1
-SLAVE = 2
+MASTER = 'master'
+SLAVE = 'slave'
+
+
+def _get_random_port():
+    import random
+    import socket
+    import struct
+
+    for _ in range(10000):
+        p = random.randint(10000, 65535)
+        # Try to open and bind the socket
+        s = socket.socket(socket.AF_INET, socket.SOCK_STREAM, socket.IPPROTO_TCP)
+        s.setsockopt(socket.SOL_SOCKET, socket.SO_REUSEADDR, 1)
+        s.setsockopt(socket.SOL_SOCKET, socket.SO_LINGER, struct.pack('ii', 1, 0))
+        try:
+            s.bind(('', p))
+            s.close()
+            return p
+        except OSError:
+            pass
 
 
 class StandardEnv(object):
-    def __init__(self, redisBinaryPath, port=6379, modulePath=None, moduleArgs=None, outputFilesFormat=None,
+    def __init__(self, redisBinaryPath, port=0, modulePath=None, moduleArgs=None, outputFilesFormat=None,
```

Comment:
You changing default to 0?

Reply:
Yes

Reply:
why?

Reply:
Because i've had situations where the tests have connected to a local Redis instead of the new Redis, if we were listening on port 6379. Perhaps the "better" way of doing this is making sure the Redis process actually remains alive after we launch it, but we have no way of verifying this; especially not with things Valgrind, MSAN, etc. where there is a slight delay between startup and actually listening/opening the port.

Reply:
This issue should be fixed by better closing the redis server and not changing the port and leave those running ..

Reply:
It's not always obvious that the redis-server is running...

Reply:
I guess I can add a --randomize-port option instead, if we want the default to still be 6379 (which I still believe is wrong for the reasons mentioned above)

Reply:
--randomize-port is better in my opinion